### PR TITLE
Add note for SessionAuthenticationStrategy in English version. #1961

### PR DESCRIPTION
**Pull request by btmaedahrs**
Please review #1961.

### CODE REVIEW

File: source_en/Security/Authentication.rst [hunk 1]
```diff
@@ -1347,6 +1347,28 @@ An audit log which records information like "When", "Who", "Which data" and "Typ
...
+    In version 1.0.x.RELEASE dependent Spring Security 3.1, \ ``org.springframework.security.web.authentication.session.ConcurrentSessionControlStrategy``\  class is provided; however,
+    it is deprecated API from Spring Security 3.2 and it is abolished API from Spring Security 4.0.
+    When upgrading version from Spring Security 3.1 to Spring Security 3.2 or later versions, changes need to be made so that it can be used with combination of following classes.
+
+    * ``ConcurrentSessionControlAuthenticationStrategy`` (added in Spring Security 3.2)
+    * ``RegisterSessionAuthenticationStrategy`` (added in Spring Security 3.2)
+    * ``org.springframework.security.web.authentication.session.SessionFixationProtectionStrategy``
+
+    For specific methods of definition,
+    refer to sample code of `Spring Security Reference -Web Application Security (Concurrency Control)- <http://docs.spring.io/spring-security/site/docs/3.2.9.RELEASE/reference/htmlsingle/#concurrent-sessions>`_.
```
Comment (ikeyat) [inline source_en/Security/Authentication.rst]: - [ ] Please correct the version in URL

**Comment by ikeyat** [comment]
Don't merge until ``translation/5.2.0`` is merged to main stream branch to avoid conflicts.

**Comment by kazuki43zoo** [comment]
@sasakitsy 

please assign this.

File: source_en/Security/Authentication.rst (hunk 1 above)
Comment (asaharas) [inline source_en/Security/Authentication.rst]: - [ ] 5.3.0 -> Spring Security 4.1.4
- [ ] 5.2.1 -> Spring Security 4.0.4

**Pull request merged**